frontend/src/styles/theme.js

```diff
@@ -1,5 +1,5 @@
 export const TIGERPATH_THEME = {
-  darkOrange: '#e49f58',
+  ECHeaderBackgroundColor: '#ec775f',
```

Comment:
In terms of organization, maybe this should be moved down next to `fallSemHeaderColor` and `springSemHeaderColor`, since they function similarly, right?